README.md

```diff
@@ -1,12 +1,53 @@
-# Northcoders News API
+# Welcome to my News API
+### Introduction
+Thank you for taking the time to view my News API. This portfolio project was made during a week-long sprint (26-8-2024 to 30-8-2024) during my [Northcoders](https://northcoders.com/) full stack software developer bootcamp. During this week I utilised the following skills in order to finish this project:
+* Javascript
+* Documentation
+* Test driven development (TDD)
+* Jest
+* Supertest
+* Express
+* Model view controller (MVC)
+* PostgreSQL
+* Fs/promises
+* Hosting using supabase for the database and render for the website
 
-In order to run this repo locally, once forked and cloned you will have to add 2 .env files to the root of the cloned repo, these files should be named .env.development and .env.test
+### Access online
+To access this API online click on the following URL: [News API](https://news-api-ovyc.onrender.com/)
 
-In the .env.development file you will need the following line PGDATABASE=nc_news and in .env.test you will need PGDATABASE=nc_news_test. 
+For a list of endpoints, simply type /api in the url after .com
 
-After adding these 2 files the repo should run locally without issue.
+To access each endpoint, you also type it in the URL after .com
 
-hosted api url = https://news-api-ovyc.onrender.com/
---- 
+### Set up
+To interact with this api locally you will need to carry out the following instructions:
+1) in your terminal run the following command in your terminal type git ```clone https://github.com/TheStringKing/News-API.git``` . This will clone the news-API repository onto your local system.
```

Comment:
"**in your terminal** run the following command **in your terminal**"
Watch out for this typo